### PR TITLE
[USING] Add Backup Best Practices

### PR DESCRIPTION
**Pull request by MaxHillebrand**
This ready for review branch separates the `Backup Best Practices` from the `Wallet Generation` chapter into its own standalone. We already have a `Password Best Practices`, and a backup is not exactly part of Wasabi specific wallet generation [i.e. you don't have to do backup for testnet].

This branch also has minor rework of the `Wallet Generation`, as I started work on this, and then realized the backup should be separated... If this should be separated into two PRs then let me know and I'll do it [I think it is alright though].

### CODE REVIEW

File: docs/using-wasabi/WalletGeneration.md [hunk 1]
```diff
@@ -23,27 +23,22 @@ The very first time you run the software the `Generate Wallet` tab will be open
 2. Name the new wallet precisely to ensure a proper differentiation from wallets created in the future.
 This label is not shared with anyone, it is only stored locally on your computer.
 
-3. Write a long and random password and _back it up_.
+3. Write a long and random password and **[back it up](/using-wasabi/BackupBestPractices)**.
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/WalletGeneration.md:26]:
```suggestion
3. Write a long and random password and **[back it up](/using-wasabi/BackupBestPractices.md)**.
```

File: docs/using-wasabi/WalletGeneration.md [hunk 2]
```diff
@@ -23,27 +23,22 @@ The very first time you run the software the `Generate Wallet` tab will be open
...
 It encrypts your secrets, and you will need it every time you want to spend bitcoin from this wallet, or recover your wallet.
 
-If you are uncertain about how to create a secure password, refer to [Password basics](/using-wasabi/WalletGeneration.md#password-basics) and [Password Best Practices](/using-wasabi/PasswordBestPractices.md) for helpful information.
+	If you are uncertain about how to create a secure password, refer to [Password basics](/using-wasabi/WalletGeneration.md#password-basics) and [Password Best Practices](/using-wasabi/PasswordBestPractices.md) for helpful information.
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/WalletGeneration.md:29]:
```suggestion
If you are uncertain about how to create a secure password, refer to [Password basics](/using-wasabi/WalletGeneration.md#password-basics) and [Password Best Practices](/using-wasabi/PasswordBestPractices.md) for helpful information.
```

File: docs/using-wasabi/WalletGeneration.md [hunk 3]
```diff
@@ -23,27 +23,22 @@ The very first time you run the software the `Generate Wallet` tab will be open
...
 
-:::danger Back up your password!
-Without the password, you cannot spend your bitcoin or recover your wallet, even if you have the recovery words (Seed phrase).
-So, triple-check that you have a proper backup!
-:::
-
-4. Click the checkbox after carefully reading the Terms and Conditions, the Privacy Policy and the Legal Issues of zkSNACKs Ltd. and Wasabi Wallet.
-
-5. Click the `Generate` button.
+	:::danger Back up your password!
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/WalletGeneration.md:32]:
```suggestion
:::danger Back up your password!
```

File: docs/using-wasabi/WalletGeneration.md [hunk 4]
```diff
@@ -23,27 +23,22 @@ The very first time you run the software the `Generate Wallet` tab will be open
...
+	Without the password, you cannot spend your bitcoin or recover your wallet, even if you have the recovery words (Seed phrase).
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/WalletGeneration.md:33]:
```suggestion
Without the password, you cannot spend your bitcoin or recover your wallet, even if you have the recovery words (Seed phrase).
```

File: docs/using-wasabi/WalletGeneration.md [hunk 5]
```diff
@@ -23,27 +23,22 @@ The very first time you run the software the `Generate Wallet` tab will be open
...
+	So, triple-check that you have a proper backup!
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/WalletGeneration.md:34]:
```suggestion
So, triple-check that you have a proper backup!
```

File: docs/using-wasabi/WalletGeneration.md [hunk 6]
```diff
@@ -23,27 +23,22 @@ The very first time you run the software the `Generate Wallet` tab will be open
...
+	:::
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/WalletGeneration.md:35]:
```suggestion
:::
```

File: docs/using-wasabi/WalletGeneration.md (hunk 2 above)
Comment (MaxHillebrand) [reply docs/using-wasabi/WalletGeneration.md:29]: NACK, this is to align the details box with the numbered list. build it and see the difference.

File: docs/using-wasabi/WalletGeneration.md [hunk 7]
```diff
@@ -23,27 +23,22 @@ The very first time you run the software the `Generate Wallet` tab will be open
...
```
Comment (yahiheb) [inline docs/using-wasabi/WalletGeneration.md:35]:
```suggestion
	:::

```

**Pull request merged**